xml/ay_partitioning.xml

```diff
@@ -2510,6 +2511,50 @@ as per ancorgs' comment on Github-->
       </varlistentry>
      </variablelist>
      
+     <para>
+       The <literal>controller_id</literal> element is required.
+     </para>
+     <para>
+       There are two optional elements, <literal>wwpn</literal> 
+       (Worldwide Port Number, the target port through which the SCSI device 
+       is attached), and <literal>fzp_lun</literal>
```

Comment:
*fzp_lun*  should be *fcp_lun*.

Reply:
Oh, I thought everything had to have a z :)